app/src/main/java/com/example/ubclaunchpad/launchnote/photoBrowser/ExpandPhotoActivity.kt

```diff
@@ -87,6 +108,27 @@ class ExpandPhotoActivity : AppCompatActivity() {
         }
     }
 
+    /**
+     * Called when toolbar's "delete" is pressed
+     */
+    private fun onDeletePressed() {
+        // get the id of the current item displayed in the ViewPager and delete it
+        LaunchNoteDatabase
+                .getDatabase(this)
+                ?.picNoteDao()
+                ?.findById(picNotes[viewPager.currentItem].id)
+                ?.map { picNotes ->
+                    picNotes.forEach {
+                        val success = PicNote.deleteFromDb(this, it)
+                        if (!success) Toast.makeText(this, "Cannot delete $defaultImageId", Toast.LENGTH_LONG).show()
+                    }
+                }?.subscribeOn(Schedulers.io())
+                ?.observeOn(AndroidSchedulers.mainThread())
+                ?.subscribe()
```

Comment:
How do you notify the activity that opened this view that the photo has been deleted. Just by looking at the code it seems that if we delete an image here, it wont get reflected on the "show all photos view"

Reply:
right now deleting a photo takes you back to AllPhotosFragment, which will reload the images.